RPIPs/RPIP-vop.md

```diff
@@ -0,0 +1,309 @@
+---
+rpip: 
+title: Voted Operational Process Framework
+description: 
+author: LongForWisdom (@LongForWisdom)
+discussions-to: 
+status: Draft
+type: Meta
+created: 2023-10-02
+---
+
+## Abstract
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
+
+$TODO
+
+## Motivation
+
+The motivation for some VOP-like framework can be summarized as the desire for increased transparency and extensibility for DAO operational decision-making.
+
+**Transparency**
+
+A VOP-like framework allows participants in the Rocket Pool DAOs to more easily be able to understand the common processes that shape the DAOs. It does this in a few main ways:
+* The types of process that exist can be listed, summarized and presented in such a way that is more-approachable to the average participant than reading through multiple full RPIPs. Participant attention is a key resource and should not be wasted on implementation details, if all that is desired is a high-level overview of 'how does process x work in Rocket Pool'.
+* Votes and discussions can be labelled explicitly according to the process they are using. This makes it easier for participants to quickly slot items into the appropriate mental boxes. An RPIP is important, because it represents a change to the status quo. A GMC election is also important, but it doesn't represent a change to how Rocket Pool works. A grants appeal is potentially unimportant to a less-engaged participant, and can be more safely ignored. 
```

Comment:
Can probably ignore this comment, with the added context of slowing my brain down, my one worry would be that a formal grants appeal process sounds like it *could* be appropriate for a VOP. Maybe so more definition on what should be considered to be a VOP is needed in this doc, I'll come back to this comment thread after I make it to the bottom.

Reply:
Ah okay, now coming back to this I think the point here is that since VOPs allow for different voter groups and voting rules, you can make it so not everyone has to pay attention to every VOP-stream. If I'm correct perhaps it's worth stating a bit more explicitly.